supertokens_python/recipe/usermetadata/interfaces.py

```diff
@@ -1,23 +1,40 @@
 from abc import ABC, abstractmethod
-from typing import Any, Dict
+from typing import Any, Dict, Literal
 
 
-class RecipeInterface(ABC):
-    def __init__(self):
-        pass
+class GetUserMetadataResult(ABC):
+    def __init__(self, status: Literal['OK'], metadata: Dict[str, Any]):
+        self.status = status
+        self.is_ok = status == 'OK'
+        self.metadata = metadata
+
+
+class UpdateUserMetadataResult(ABC):
+    def __init__(self, status: Literal['OK'], metadata: Dict[str, Any]):
+        self.status = status
+        self.is_ok = status == 'OK'
+        self.metadata = metadata
```

Comment:
These two are identical. You can just make them into one class called `MetadataResult`